Lib/test/test_dis.py

```diff
@@ -1050,12 +1105,12 @@ def test_explicit_first_line(self):
         self.assertEqual(list(actual), expected_opinfo_outer)
 
     def test_source_line_in_disassembly(self):
-        # Use the line in the source code
-        actual = dis.Bytecode(simple).dis()[:3]
+        # Use the line in the source code (split extracts the line no)
+        actual = dis.Bytecode(simple).dis().split(" ")[0]
```

Comment:
Or just use `[:3]`.

Or use `[:4]` and add a space to the expected value (this is more robust).
```
actual = dis.Bytecode(simple).dis()[:4]
expected = "{:>3} ".format(simple.__code__.co_firstlineno)
```